src/scripts/ping-okta-server.ts

```diff
@@ -0,0 +1,117 @@
+import fetch from 'node-fetch';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as E from 'fp-ts/lib/Either';
+import { RootCommand } from '..';
+import * as Console from 'fp-ts/lib/Console';
+import { pipe } from 'fp-ts/lib/function';
+import { Argv } from 'yargs';
+
+export const pingOktaServer = async (
+  clientId: string,
+  organisationUrl: string
+): Promise<TE.TaskEither<Error, string>> => {
+  const response = await TE.chain(
+    ({
+      organisationUrl,
+      clientId,
+    }: {
+      readonly organisationUrl: string;
+      readonly clientId: string;
+    }) =>
+      TE.tryCatch(
+        async () => {
+          return await fetch(
+            `${organisationUrl}/oauth2/default/.well-known/oauth-authorization-server?client_id=${clientId}`
+          );
```

Comment:
**question:** how are the secrets tested by this call?

Reply:
i used client.oauth.getAccessToken like you suggessted in commit: d1813205e293bc0a35448174005ac4d7501895b2 
I should've looked for something like this in the sdk. I was only searching for an api before